sets/sets-eng.svg

```diff
@@ -0,0 +1,1466 @@
+         sodipodi:role="line"
+         x="202.18823"
+         y="554.97113"
+         id="tspan5546"> </tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="569.14612"
+         id="tspan5549"> </tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="583.32111"
+         id="tspan5551">Add element k</tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="597.49609"
+         id="tspan5553">Remove element (<tspan
+   style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-family:'Oxygen Mono';-inkscape-font-specification:'Oxygen Mono'"
+   id="tspan5557">KeyError</tspan> if k is not in the set)</tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="611.67114"
+         id="tspan5555">Remove k element only if it's in the set</tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="625.84613"
+         id="tspan5563">It will remove and show any element</tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="640.02112"
+         id="tspan5588"> </tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="654.19611"
+         id="tspan5590"> </tspan><tspan
+         style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:11.25px;line-height:1.25999999;font-family:Oxygen;-inkscape-font-specification:Oxygen"
+         sodipodi:role="line"
+         x="202.18823"
+         y="668.37109"
+         id="tspan5592">True if sets are disjoint? (tj. they don't have any</tspan><tspan
```

Comment:
`tj.` is Czech, use `e.g.`

Reply:
use [i.e.](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/)